python/test/unit/language/test_core.py

```diff
@@ -3040,6 +3040,8 @@ def test_dot(M, N, K, num_warps, col_a, col_b, epilogue, input_precision, in_dty
             pytest.skip("kpack too large for K")
         if not is_hip() and kpack == 2:
             pytest.skip("Skip duplicated tests on nv path")
+        if not is_hip() and (M, N, K) == (16, 16, 8):
```

Comment:
As far as this feature is targeted on gfx9 arch, could you please also skip it for Navi31 arch?

arch name could be taken with 
``triton.runtime.driver.active.get_current_target().arch``

Reply:
I did that, but I abstracted that incantation into a function